### DISCUSSION TITLE
Is there any way to POST data/changes rather than just fetch different information views?

### DISCUSSION THREAD

**Discussion by kennedyjosh** [cwendt94/espn-api#602] [Q&A]
Curious if you’ve looked into this at all. For instance, setting lineup or add/dropping a player via this API. It is something I am interested in myself, to maybe make a custom one-stop interface for ESPN fantasy football

**Comment by cwendt94** [comment]
I haven't looked into it too much but definitely think its possible. Just from look at some of the API calls from ESPN's website this is adding a player into your line up. I think the current authentication would still work.
API Call
```
POST 
https://lm-api-writes.fantasy.espn.com/apis/v3/games/ffl/seasons/2024/segments/0/leagues/1436883837/transactions/

PAYLOAD
{
  "isLeagueManager":false,
  "teamId":9,"
  type":"ROSTER",
  "memberId":"{SWID}",
  "scoringPeriodId":9,
  "executionType":"EXECUTE",
  "items":[{"playerId":3045147,"type":"LINEUP","fromLineupSlotId":20,"toLineupSlotId":2
    {"playerId":4429795,"type":"LINEUP","fromLineupSlotId":2,"toLineupSlotId":20}]
}
```
It would probably take a little work mapping the correct values and setting up the different API calls for the different actions.
